src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -1922,6 +1922,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Friend Overrides Function GetUsedAssemblyReferences(Optional cancellationToken As CancellationToken = Nothing) As ImmutableArray(Of MetadataReference)
+            Throw New System.NotImplementedException()
```

Comment:
Is this planned on being implemented for VB?

Reply:
> Is this planned on being implemented for VB?

Yes

---
In reply to: [347633548](https://github.com/dotnet/roslyn/pull/39807#discussion_r347633548) [](ancestors = 347633548)